server/app/views/applicant/ProgramIndexTemplate.html

```diff
@@ -44,6 +44,18 @@
         <h1
           th:text="${isGuest}? #{content.saveTimeServices} : #{content.findPrograms}"
         ></h1>
+
+        <!-- This would go into the css file -->
+        <style type="text/css">
+          .custom-red-color {
+            filter: brightness(0) saturate(100%) invert(14%) sepia(97%)
+              saturate(7463%) hue-rotate(2deg) brightness(95%) contrast(115%);
+          }
+        </style>
+
+        <!-- Just provide the icon type -->
+        <cf:icon type="https" class="mb-6 h-8 w-8 custom-red-color" />
```

Comment:
This has the tag processor configured to build the image name in the format of `icon-resetofname.svg`. It will handle adding the relative path and dealing with the asset finder. 

It will throw if the image isn't found to prevent trying to using an image that is spelled wrong or doesn't exist.

We could pass the type value as `icon-https`, that is the file name without the extension, but I don't want to leave it open to free form full paths. It's redundant and error prone.